handlers/s3client.go

```diff
@@ -48,3 +51,28 @@ func StreamObjectFromS3(client *AwsClient, bucket string, key string) ([]byte, e
 	return buff.Bytes(), nil
 
 }
+
+// GetObjectSize returns object size in bytes
+func GetObjectSize(client *AwsClient, bucket string, key string) (*int64, error) {
+	svc := s3.New(client.session)
+	input := &s3.HeadObjectInput{
+		Bucket: aws.String(bucket),
+		Key:    aws.String(key),
+	}
+
+	result, err := svc.HeadObject(input)
+	if err != nil {
```

Comment:
I'm confused what this block is doing, couldn't this just be
```go
fmt.Println(err.Error())
return nil, err
```
or does casting it to `awserr.Error` actually return a different message?

Reply:
Errors returned by the SDK are backed by a concrete type that will satisfy the awserr.Error interface.

https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/handling-errors.html